support-workers/src/main/scala/com/gu/support/workers/lambdas/SendThankYouEmail.scala

```diff
@@ -44,19 +44,32 @@ class SendThankYouEmail(
             response <- services.paperRoundService.agents()
           } yield response.data.agents.find(_.refId == id)
       }
+
     val emailBuilder = new EmailBuilder(
       services.zuoraService.getMandateIdFromAccountNumber,
       getAgentDetails,
       services.promotionService,
       Configuration.stage,
     )
 
-    for {
-      emailFields <- emailBuilder.buildEmail(state)
-      emailResult <- Future.sequence(emailFields.map(emailService.send))
-    } yield HandlerResult(emailResult, requestInfo)
+    /** We generate these users `testGuardianDoNotUse` as part of our e2e testing and don't want to create a braze user
```

Comment:
What does `testGuardianDoNotUse` refer to here? Is/was that a pattern we were going to use for the email?

Reply:
My bad - it was the old address I was using that I didn't like the look of. Changed.